packages/ai/src/ai/modules/remote/remote.py

```
@@ -8,17 +8,15 @@ class Pipe:
     Define our class to keep track of a processing pipe.
     """
 
-    def __init__(self, apikey: str = '', loader: Any = None, input: List[str] = [], output: List[str] = [], usage: Dict[str, int] = {}):
+    def __init__(self, apikey: str = '', loader: Any = None, input: Optional[List[str]] = None, output: Optional[List[str]] = None, usage: Optional[Dict[str, int]] = None):
         """
         Сreate an instance of the pipe context.
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `С` (CYRILLIC CAPITAL LETTER ES). Did you mean `C` (LATIN CAPITAL LETTER C)?

(RUF002)